lib/matplotlib/_mathtext.py

```diff
@@ -2168,6 +2172,18 @@ def __init__(self):
             - (p.required_group | Error("Expected \\overline{value}"))
         )
 
+        p.overset <<= Group(
+            Suppress(Literal(r"\overset"))
+            - ((p.required_group + p.symbol_group)
```

Comment:
Should this not just use simple_group instead?  (in which case symbol_group could go away)  Looks like TeX is perfectly happy with `\overset{ab}{}` or `\overset{ab}{cd}` (you may need to add tests that both "overset over nothing" and "overset over multiple characters" work).

Reply:
TeX is also happy with `\overset{}{}`, this made me _remove_ the tests (and also convert the `required_group` into a `simple_group`), since we only test failures:
https://github.com/matplotlib/matplotlib/blob/739898e708ec1dc8228b421e3bfba74a631706dc/lib/matplotlib/tests/test_mathtext.py#L236-L238